knoema/__init__.py

```diff
@@ -60,24 +60,68 @@ def get(dataset = None, include_metadata = False, mnemonics = None, transform =
         columns = columns.split(';')
 
     frequency = None
-    timerange = None
+    time = None
+    time_params_count = 0
     has_agg = False
+    advanced = False
     for name, value in dim_values.copy().items():
         if is_equal_strings_ignore_case(name, 'frequency'):
             frequency = value
             del dim_values[name]
             continue
+
         if is_equal_strings_ignore_case(name, 'transform'):
             transform = value
             del dim_values[name]
             continue
+
         if is_equal_strings_ignore_case(name, 'timerange'):
-            timerange = value
+            time = value
+            del dim_values[name]
+            if time != None: 
+                dim_values['timerange'] = time
+                ++time_params_count
+            continue
+
+        if is_equal_strings_ignore_case(name, 'timesince'):
+            time = value
+            del dim_values[name]
+            if time != None: 
+                dim_values['timesince'] = time
+                advanced = True
+                ++time_params_count
+            continue
+
+        if is_equal_strings_ignore_case(name, 'timelast'):
+            time = value
+            del dim_values[name]
+            if time != None: 
+                dim_values['timelast'] = time
+                advanced = True
+                ++time_params_count
+            continue
+        
+        if is_equal_strings_ignore_case(name, 'timemembers'):
+            time = value
+            del dim_values[name]
+            if time != None: 
+                dim_values['timemembers'] = time
+                advanced = True
+                ++time_params_count
             continue
 
         if len(value) > 0 and value[0].startswith('@'):
             has_agg = True
 
+    if not dataset:
+        raise ValueError('Dataset id is not specified')

```

Comment:
if you search by mnemonics without dataset specified you get this error
`data_frame = knoema.get(mnemonics = 'mnemonic1;mnemonic2')`

Reply:
+